x-pack/test/performance/scalability/config.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrConfigProviderContext } from '@kbn/test';
+import fs from 'fs';
+import path from 'path';
+import { serializeApmGlobalLabels } from '../utils';
+import { ScalabilityTestRunner } from './runner';
+
+// These "secret" values are intentionally written in the source.
+const APM_SERVER_URL = 'https://142fea2d3047486e925eb8b223559cae.apm.europe-west1.gcp.cloud.es.io';
+const APM_PUBLIC_TOKEN = 'pWFFEym07AKBBhUE2i';
```

Comment:
:warning: Using kibana-stats cluster here, as we discussed on sync meeting we need to create a new internal cluster.